core/cautils/sliceutils.go

```diff
@@ -0,0 +1,27 @@
+package cautils
+
+import "github.com/kubescape/opa-utils/reporthandling"
+
+func IsSubSliceScanningScopeType(haystack []reporthandling.ScanningScopeType, needle []reporthandling.ScanningScopeType) bool {
```

Comment:
What is this function?

Reply:
I'll remove it, it is not used now.(I thought to keep it since it is already tested)